ci-operator/config/openshift/installer/openshift-installer-release-4.5.yaml

```diff
@@ -189,6 +189,10 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-metal-assisted
+  steps:
+    cluster_profile: packet
+    workflow: baremetalds-assisted
```

Comment:
You'll probably want this to go to master too, not just 4.5.

Reply:
At the moment we support only 4.5 release. But sure it will be added later.

Reply:
You generally can't make changes to 4.5 directly, everything goes through 4.6/master at this point. So if you want the ability to test PR's, you'll need it on master. It's fine to enable it on master even if it doesn't work.

Reply:
@YuviGold why do you want to use the `openshift/installer` repo instead of your own repo to execute the `baremetalds-assisted` worklow?

Reply:
> You generally can't make changes to 4.5 directly, everything goes through 4.6/master at this point. So if you want the ability to test PR's, you'll need it on master. It's fine to enable it on master even if it doesn't work.

@stbenjam The point of inserting this workflow is to be tested as part of the installer.
Right now it doesn't work for 4.6 so this PR will just be hanging for a long time.
I prefer to create a new PR when it will be relevant for the assisted-installer.

> @YuviGold why do you want to use the `openshift/installer` repo instead of your own repo to execute the `baremetalds-assisted` worklow?

@andfasano At the moment the repo is not a part of the OpenShift GitHub organization.
I will add the appropriate config file and Prow job when it will be possible.

Reply:
> @stbenjam The point of inserting this workflow is to be tested as part of the installer.
Right now it doesn't work for 4.6 so this PR will just be hanging for a long time.
I prefer to create a new PR when it will be relevant for the assisted-installer.

I still don't understand the testing strategy, what are you wanting to do with this job then?

The way you have it, presubmits are for PR's. And since you only have it in 4.5, you'll only be able to run the assisted install tests on code that's already merged to master.